plugin/entry.js

```diff
@@ -2,14 +2,13 @@
 
 // Add out component to window.__STYLEGUIDE_PLUGIN_COMPONENTS_DO_NOT_TOUCH__ so the styleguide client can pick them
 // up
-window.__STYLEGUIDE_PLUGIN_COMPONENTS_DO_NOT_TOUCH__ = window.__STYLEGUIDE_PLUGIN_COMPONENTS_DO_NOT_TOUCH__ || {};
-window.__STYLEGUIDE_PLUGIN_COMPONENTS_DO_NOT_TOUCH__['%%path%%'] = {
+window.__STYLEGUIDE_PLUGIN_CLIENT_API.loadComplete('%%path%%', {
   name: '%%name%%',
   meta: require('!!%%metaLoader%%!%%requestPath%%'),
   source: require('!!%%sourceLoader%%!%%requestPath%%'),
   path: '%%path%%',
-  component: function component() {
+  component: (function component() {
```

Comment:
@jantimon I thought for some reason we wanted to avoid calling this here. I guess it's necessary due the new setup.